recipes/dlib/lapack_sgetrf.patch

```diff
@@ -0,0 +1,20 @@
+--- dlib/cmake_find_blas.txt
++++ dlib/cmake_find_blas.txt
+@@ -140,15 +140,8 @@
+          set(blas_found 1)
+          message(STATUS "Found OpenBLAS library")
+          set(CMAKE_REQUIRED_LIBRARIES ${blas_libraries})
+-         # If you compiled OpenBLAS with LAPACK in it then it should have the
```

Comment:
@davisking We control the OpenBLAS build so if it's broken then we can fix it/that is a bug. Are there any parts of dlib exposed via the Python API that make heavy use of BLAS/LAPACK?